flocker_bb/builders/maint.py

```diff
@@ -232,7 +232,10 @@ def _destroy_cloud_volumes(self, volumes):
         Unconditionally and irrevocably destroy all of the given cloud volumes.
         """
         for volume in volumes:
-            volume.destroy()
+            try:
+                volume.destroy()
+            except:
+                pass
```

Comment:
We could easily log it to the buildbot logs.
Getting it on the step log would be more involved, and I'm not really sure it is worth it, given that we are throwing this away soon, hopefully.